src/main/java/tom/TaskList.java

```diff
@@ -0,0 +1,56 @@
+package tom;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Represents the list of tasks to be done by the user.
+ */
+public class TaskList {
+    private ArrayList<Task> ls;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.ls = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
+        return ls;
+    }
+
+    public void list() throws IOException {
+        Ui.list(ls);
+        Storage.writeLines(ls);
+    }
+
+    public void add(Task task) throws IOException {
+        ls.add(task);
+    }
+
+    public void delete(int n) throws IOException {
+        Task removed = ls.get(n-1);
+        ls.remove(removed);
+        Ui.delete(removed, ls);
+    }
+
+    public void mark(int n) throws IOException {
+        Task cur = ls.get(n-1);
+        cur.Mark();
+        Ui.mark(cur);
+    }
+
+    public void unmark(int n) throws IOException {
+        Task cur = ls.get(n-1);
```

Comment:
I think there should be one space on each side of the operator:
```suggestion
        Task cur = ls.get(n - 1);
```